server/app/views/admin/programs/ProgramFormBuilder.java

```diff
@@ -382,7 +382,8 @@ static DivTag buildApplicationStepDiv(
             .setMarkdownSupported(true)
             .setValue(descriptionValue);
 
-    if (indexPlusOne.equals("1")) {
+    Boolean isRequired = indexPlusOne.equals("1") && !isCommonIntakeForm;
```

Comment:
Functionality change (2/2): before, '*' would appear when editing a pre-screener

This is tested by including whether the span containing '*' is disabled in `expectFormFieldDisabled`